pypeit/data/utils.py

```diff
@@ -23,8 +23,8 @@
 
 __all__ = ['Paths', 'load_telluric_grid', 'load_thar_spec',
            'load_sky_spectrum', 'get_reid_arxiv_filepath',
-           'get_skisim_filepath', 'get_sensfunc_filepath',
-           'fetch_remote_file', 'write_file_to_cache']
+           'get_skisim_filepath', 'get_spectrograph_gain_ronoise_dirpath',
```

Comment:
See comment attached to function, but this is not needed for your use case.

Reply:
Removed